core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -382,14 +383,14 @@ object ConfigCommand extends Config {
         case _ =>
           println(s"Configs for ${entityType.dropRight(1)} ${entity} are:")
       }
-      getConfig(adminClient, entityType, entity, includeSynonyms = true).foreach { entry =>
+      getConfig(adminClient, entityType, entity, includeSynonyms = true, describeAll).foreach { entry =>
         val synonyms = entry.synonyms.asScala.map(synonym => s"${synonym.source}:${synonym.name}=${synonym.value}").mkString(", ")
         println(s"  ${entry.name}=${entry.value} sensitive=${entry.isSensitive} synonyms={$synonyms}")
       }
     }
   }
 
-  private def getConfig(adminClient: Admin, entityType: String, entityName: String, includeSynonyms: Boolean): Seq[ConfigEntry] = {
+  private def getConfig(adminClient: Admin, entityType: String, entityName: String, includeSynonyms: Boolean, describeAll: Boolean) = {
```

Comment:
The `describeAll` argument is no longer being used. Maybe we lost some logic when rebasing?

Reply:
Yes unfortunately it did. Thanks @hachikuji for catching this. Will update shortly.